handbook/customer-first.md

```diff
@@ -2,16 +2,19 @@
 
 ### Next steps after a customer conversation
 After a customer conversation, it can sometimes feel like there are 1001 things to do, but it can be hard to know where to start.  Here are some tips:
-### For customer requests
+
+##### For customer requests
 - Locate the appropriate issue, or create it if it doesn't already exist.  (To avoid duplication, be creative when searching GitHub for issues- it can often take a couple of tries with different keywords to find an existing issue.)
 - Is the issue clear and easy to understand, with appropriate context?  (Default to public: declassify into public issues in fleetdm/fleet whenever possible)
 - Is there a key date or timeframe that the customer is hoping to meet, please note that in the the issue.
 - Make sure the issue has a "customer request" label.
 - Post in #g-product with a link to the issue to draw extra attention to the customer request so it is visible ASAP for Fleet's product team.
 - Have we provided a link to that issue for the customer to remind everyone of the plan, and for the sake of visibility, so other folks who weren't directly involved are up to speed?  (e.g. "Hi everyone, here's a link to the issue we discussed on today's call: […link…](https://omfgdogs.com)")
 
+##### Generating a trial license key
```

Comment:
```suggestion
#### Generating a trial license key
```
Same as above, I'll make an issue about adding a section to the handbook about headings on docs and handbook pages.